common/static/common/templates/discussion/thread-response.underscore

```diff
@@ -1,12 +1,10 @@
 <div class="discussion-response"></div>
 <a href="#" class="action-show-comments">
     <%
-    var fmts = ngettext(
-        "Show Comment (%(num_comments)s)",
-        "Show Comments (%(num_comments)s)",
-        comments.length
-    );
-    print(interpolate(fmts, {num_comments: comments.length}, true));
+    var fmts = ngettext("Show Comment (%(num_comments)s)", "Show Comments (%(num_comments)s)", comments.length);
```

Comment:
Why did you make the change to put this on one line? Is it necessary? I think it's more readable separated out